subsys/usb/device/usb_device.c

```diff
@@ -146,6 +148,9 @@ struct usb_setup_packet_packed {
 	uint16_t wLength;
 } __packed;
 
+/* forward declaration of static functions */
```

Comment:
Not addressed, please do not comment on obvious.

Reply:
Removed